templates/topics.html

```diff
@@ -19,7 +19,7 @@
 	{% endif %}
 
 	<form action="/topics" method="POST">
-		<div class="columns is-mobile">
```

Comment:
I think this should _help_ but how well does this work in practice, or should we just look for a secondary interface element to use on mobile?

Reply:
I did not add the `is-mobile` class, so I left it there. However, it wasn't working so I added a CSS media query to change how the list elements are rendered.